bio/vg/autoindex/giraffe/wrapper.py

```
@@ -0,0 +1,22 @@
+__copyright__ = "Copyright 2024, Felix Mölder"
+__email__ = "felix.moelder@uk-essen.de"
+__license__ = "MIT"
+
+from snakemake.shell import shell
+from os import path
+
+# Extract arguments.
+extra = snakemake.params.get("extra", "")
+log = snakemake.log_fmt_shell(stdout=False, stderr=True)
+
+prefix_path = path.splitext(snakemake.output[0])[0]
```

Ruff

Undefined name `snakemake`

(F821)